### PR TITLE
normalize module name & introduce metadata

### PR DESCRIPTION
**Pull request by igalic**
after normalizing the name, we also have to update .sync.yml's secret!

n.b.: this will probably not build, because it badly needs a modulesync

### CODE REVIEW

**Comment by igalic** [comment]
another step towards https://github.com/voxpupuli/puppet-amanda/issues/28

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "puppet-amanda",
+  "version": "1.0.3",
+  "source": "https://github.com/voxpupuli/puppet-amanda",
+  "author": "voxpupuli",
+  "license": "Apache License 2.0",
```
Comment (bastelfreak) [inline metadata.json]: shouldn't that be Apache-2.0?

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "summary": "Amanda Module for Puppet",
+  "description": "Amanda Module for Puppet",
+  "project_page": "https://github.com/voxpupuli/puppet-amanda",
+  "issues_url": "https://github.com/voxpupuli/puppet-amanda/issues",
+  "dependenies": [
```
Comment (juniorsysadmin) [inline metadata.json]: typo

File: metadata.json (hunk 1 above)
Comment (juniorsysadmin) [inline metadata.json]: Is this a valid license string?

**Comment by igalic** [comment]
thanks @bastelfreak & @juniorsysadmin ! i didn't run tests locally, because bundle install failed D:

**Comment by bastelfreak** [comment]
This is now obsolete.

**Pull request closed without merging**